activerecord/lib/active_record/relation.rb

```diff
@@ -494,7 +494,7 @@ def destroy(id)
 
     # Deletes the records without instantiating the records
     # first, and hence not calling the {#destroy}[rdoc-ref:Persistence#destroy]
-    # method nor invoking callbacks.
+    # method is not invoking callbacks.
```

Comment:
Hi @denqxotl if you read this in continuity, the current sentence is correct, and new change is not.